assets/css/style.css

```diff
@@ -115,6 +152,10 @@ a {
     text-align: center;
 }
 
+.about-me-title h2, .my-work-title h2, .my-contact-title h2 {
+    font-size:medium;
```

Comment:
if possible, try to use rem or px when referring to font size.

Reply:
thanks Dom